tests/suites/spaces_ec2/util.sh

```diff
@@ -8,7 +8,7 @@ add_multi_nic_machine() {
 
 	# Ensure machine is deployed to the same az as our nic
 	az=$(aws ec2 describe-network-interfaces --filters Name=network-interface-id,Values="$hotplug_nic_id" | jq -r ".NetworkInterfaces[0].AvailabilityZone")
-	juju add-machine --constraints zones="${az}"
+	juju add-machine --constraints zones="${az}" --base ubuntu@24.04
```

Comment:
That's fine. There will always be some leg work when a new LTS comes out, otherwise we're not exercising or testing it. Pinning everything to an older LTS ensures we're not testing the new one.

Reply:
Sure thing. I'll drop